src/js/media-live-button.js

```diff
@@ -35,10 +35,10 @@ slotTemplate.innerHTML = `
   </style>
 
   <slot name="indicator">${indicatorSVG}</slot>
-  <!-- 
```

Comment:
I believe this is an HTML comment on purpose and would break things if it's removed. Though, it would be very nice if it wasn't needed anymore, which could also be the case. In which case, it can be removed altogether.

Reply:
it's about these 2 elements `<slot name="spacer">&nbsp;</slot><slot name="text">LIVE</slot>`
they shouldn't have a line break between them

removing the comment in runtime is fine I think